routes/index.js

```diff
@@ -1,19 +1,20 @@
 const express = require('express');
 const router = express.Router();
 const fs = require('fs');
-const mongoConnection = process.env.MONGODB_URI || 'mongodb://localhost:27017/faq';
+const mongoConnection = 'mongodb://127.0.0.1:27017/faq';
```

Comment:
Changing this line will break the app for your group members, and prevent the use of `process.env.MONGODB_URI` to specify the connection.

I'd recommend leaving it as it was. Then to get your database connection working, you need to set the `MONGODB_URI` before you run the script by starting the server like this:

```
MONGODB_URI=127.0.0.1:27017/faq npm start
```

Try that and see if it works for you.

Reply:
It broke again!